mungegithub/README.md

```diff
@@ -0,0 +1,109 @@
+  - You can cancel "lgtm" by commenting `/lgtm cancel`, if you `/lgtm` before.
+
+![alt tag](https://github.com/istio/test-infra/blob/mungegithub_doc/mungegithub/images/lgtm.png)
+
+* **approved**
+
+  - When a person's name is in all OWNERS file(s) which is/are able to cover all changed files, he/she is a valid approver.
+A valid approver is able to say `/approve` and mungegithub will add "approved" label.  
+  - We already got ride of `/approve no-issue`, every PR can be approved by just saying `/approve`  
+  - A valid approver can say `/approve` to his/her own PR  
+  - You can cancel "approved" by commenting "/approve cancel", if you `/approve` before.
+
+![alt tag](https://github.com/istio/test-infra/blob/mungegithub_doc/mungegithub/images/approve.png)
+
+* **release-note**
+  - Use [PR template](https://github.com/istio/istio/blob/master/.github/PULL_REQUEST_TEMPLATE.md) to add release-note, make sure add "none" if that's not necessary for this PR.
+  
+  ![alt tag](https://github.com/istio/test-infra/blob/mungegithub_doc/mungegithub/images/release-note-template.png)
+  
+  - You can also leave a comment `/release-note-none` to set it NONE
+  > When all requirements are satisfied except "release-note", mungegithub will add "do-not-merge" label to force you clear release-note. Please note, when you fix release-note, you have to manually remove that label after you make sure everything looks good.
+  ![alt tag](https://github.com/istio/test-infra/blob/mungegithub_doc/mungegithub/images/do-not-merge-release-note.png)
+  
+* **do-not-merge**
+  - "do-not-merge" label will block merge anytime. Add it if you want to temporarily block merge.
+  
+### Stage Two
+  When a PR satisfies all requirements in stage one, it will be picked up by submit-queue. Submit-queue is going to sequentially retest each pr by asking prow to rerun all required tests.
+  ![alt tag](https://github.com/istio/test-infra/blob/mungegithub_doc/mungegithub/images/retest.png)
+  - Idealy, you don't need to update branch (also called rebase/merge from master) before run tests because prow will merge your branch into master locally and run tests against this code. You will not see any changes happens on your branch but you can find some clue when you go to prow log.
+  ![alt tag](https://github.com/istio/test-infra/blob/mungegithub_doc/mungegithub/images/auto-rebase-log.png)
+  - Use `/test all` or `/retest`, you can manually verify and retest everything.
+  
+ ### Stage Three
+   If a PR passes all required tests in rerun, then this PR is good to go and will be auto merged by submit-queue, wohoo!
+   ![alt tag](https://github.com/istio/test-infra/blob/mungegithub_doc/mungegithub/images/merge.png)
+ 
+ 
+ ## FAQ
+ 
+   * Why my pr hasn't been retest
+   - Four kinds of labels
+   - No "do-not-merge" label
+   - All required checks pass
+   - Maybe another PR is being retested, yours are waiting in the queue
+
+   * Why I said `/lgtm`, it also adds "approved" label
```

Comment:
-> Why does my `/lgtm` adds both "lgtm" and "approved" labels?
Also the star was not rendered to a bullet. Maybe try removing the preceding new line?